src/pages/Login/index.tsx

```diff
@@ -119,7 +120,37 @@ const LoginPage = ({ type }: ILoginPage): JSX.Element => {
       content = (<BusinessForm onSubmit={onSubmit}/>);
       break;
     case 'login':
-      content = null;
+      onSubmit = async (data: LoginFormData) => {
+        const email = data.email;
+        const password = data.password;
+        signInWithEmailAndPassword(auth, email, password)
+          .then(async (userCredential) => {
+            // Signed in
+            const user = userCredential.user;
+            const reqData = {
+              firebaseId: user.uid,
+              email: data.email,
+              password: data.password,
+            };
+            await fetch(`/business/${data.businessId}/user`, {
+              method: 'POST',
```

Comment:
Why the POST method? You are not creating anything. You are getting the employee information from the database after a successful login.